http/handler.go

```diff
@@ -168,20 +142,18 @@ func (i internalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	reqLogEnt := i.ctx.ReqLog.Add(req)
-	defer i.ctx.ReqLog.Finish(reqLogEnt)
-
-	//ps: take note of the name clash - commands.Context != context.Context
-	req.SetInvocContext(i.ctx)
+	if req.Context == nil {
+		fmt.Println("cmd.Call: request nil")
```

Comment:
request is nil? or request context is nil?

Reply:
Sorry, that was an accident. Ignore the whole if block.